frontend/src/components/VisaStatusOverviewList/OverviewContent.js

```diff
@@ -0,0 +1,243 @@
+import React, { useMemo, useState } from "react";
+import PropTypes from "prop-types";
+import { useSelector } from "react-redux";
+import { Row, Col, Button, Tooltip, Space, Collapse, Tag, Modal } from "antd";
+import { MailOutlined, QqOutlined, EllipsisOutlined, PlusOutlined } from "@ant-design/icons";
+import { useTranslation } from "react-i18next";
+import { useScreenXS } from "../../hooks";
+import { makeNewestVisaStatusSelector } from "../../redux/selectors";
+import { findEmbassyAttributeByCode } from "../../utils/USEmbassy";
+import { overviewAttrProps, newestOverviewProps, overviewProps } from "./overviewPropTypes";
+import "./VisaStatusOverview.less";
+
+const { Panel } = Collapse;
+
+const QQTGSubsModalHook = (qq, tg) => {
+    const { t } = useTranslation();
+    const [isModalVisible, setIsModalVisible] = useState(false);
+    return [
+        <Modal
+            title={t("QQTGModalTitle")}
+            visible={isModalVisible}
+            onOk={() => setIsModalVisible(false)}
+            onCancel={() => setIsModalVisible(false)}
+        >
+            <p>
+                {t("QQTGModalContentTG")}
+                <a href={tg} target="_blank" rel="noreferrer">
+                    {tg}
+                </a>
+            </p>
+            <p>{t("QQTGModalContentQQDesc")}</p>
+            {qq.map((content, index) => (
+                <p>
+                    {t("QQTGModalContentQQ", { index: index + 1 })}
+                    {content}
+                </p>
+            ))}
+        </Modal>,
+        setIsModalVisible,
+    ];
+};
+
+const ContentActions = ({ children, embassyCode, onEmailSubsClick, onAddtionClick }) => {
+    const { t } = useTranslation();
+    const info = useSelector(state => state.metadata.qqTgInfo);
+    const embassyLst = useSelector(state => state.metadata.embassyLst);
+    const region = findEmbassyAttributeByCode("region", embassyCode, embassyLst);
+    const index = region === "DOMESTIC" ? "domestic" : "nonDomestic";
+    const [QQTGSubsModal, setQQTGSubsModalVisible] = QQTGSubsModalHook(info.qq[index], info.tg[index]);
+    const isVisaTypeF = useSelector(state => state.visastatusTab) === "F";
+    return (
+        <Space direction="horizontal">
+            <Tooltip title={t("overviewEmailIcon")}>
+                <Button icon={<MailOutlined />} shape="circle" onClick={() => onEmailSubsClick()} />
+            </Tooltip>
+            {isVisaTypeF && (
+                <Tooltip title={t("overviewQQIcon")}>
+                    <Button icon={<QqOutlined />} shape="circle" onClick={() => setQQTGSubsModalVisible(true)} />
+                </Tooltip>
+            )}
+            <Tooltip title={t("overviewAddtionalIcon")}>
+                <Button icon={<EllipsisOutlined rotate={90} />} shape="circle" onClick={() => onAddtionClick()} />
+            </Tooltip>
+            {isVisaTypeF && QQTGSubsModal}
```

Comment:
Mark this line, if anything we should put this out like

```js
<>
    <Space>
        ...
    </Space>
    { isVisaTypeF && QQTGSubsModal }
</>
```

Here is [the documentation for `React.Fragments`](https://reactjs.org/docs/fragments.html) and `<></>` is the short syntax of it.